acsoo/makepot_script

```diff
@@ -4,9 +4,16 @@
 
 import base64
 import os
+import re
 import subprocess
 from os.path import join
 
+
+LINE_PATTERNS_TO_REMOVE = [
+    r"(\")POT-Creation-Date.*[\s\S]",
+    r"(\")PO-Revision-Date.*[\s\S]",
```

Comment:
or `r'"PO-Revision-Date: .*?"'`, `.*?` beeing non greedy and will stop at the first "